### ISSUE TITLE
Prevent Users from Accessing "System" Page in Web UI

### ISSUE THREAD

**Issue by williamtrelawny** [Graylog2/graylog2-server#15557]
MSP's providing GLaaS may not want their customers to see any information outside their log data (incl. reports, alerts, & dashboards). Especially the information under the System menu that is strictly for system admins and not end users.

Currently, users with the Reader role (the most least-privileged role available at this time) can see the Overview, Configurations, Nodes (including the Details and Metrics pages), and Inputs pages under the System top-level menu. They also have access to the API browser, which I understand they lack the proper authorization to run most commands, but it would be nice if this page was inaccessible to end users entirely.

Even non-MSP customers who want to restrict access to system level data to only their admins and their log data to true read-only analyst users would benefit from this increased segregation.

## What?
Basically, I want to allow customers to streamline their read-only analysts' UX by limiting their exposure to only the data they need to know about. Ideally, the "System" menu in the UI and the API Browser would be totally inaccessible to them. 

## Why?
ref: HS-1628469824

Improved UX for true read-only users & analysts, who don't care or need to see Graylog admin data.

## Your Environment

* Graylog Version: 5.0.x

**Comment by ed-graylog** [comment]
This is also tied to hs-1906763315. One particular option is to create some additional permissions which remove "system" UI access and assign them to a role such as "minimal" similar to how we already have "sidecar" or "reader"
